scripts/jenkins/Makefile.jenkins

```diff
@@ -445,6 +445,16 @@ test-kerberos-hadoop-3-hdp: test-hadoop-3-smoke-standalone test-hadoop-common-sm
 
 test-kerberos-hadoop-3-spnego: test-kerberos-hadoop-3-standalone test-hadoop-common-smoke-hive-spnego
 
+test-kerberos-verify-token-refresh: h2o_one_node
+	cat h2o_one_node | grep job | sed 's/job/application/g' > h2o_yarn_id
+	@if [ -z "`cat h2o_yarn_id`" ]; then\
+		echo "YARN Application ID not found."; \
+		exit 1; \
+	fi
+	@echo "YARN Application ID is `cat h2o_yarn_id`"
+	-yarn application -kill `cat h2o_yarn_id`
+	yarn logs -applicationId `cat h2o_yarn_id` | grep -e "TOKEN REFRESH: Updating credentials"
```

Comment:
does it check anywhere that the pattern was found?

Reply:
if the pattern was not found grep returns non-zero exit status which is detected by make as failure of the target

Reply:
duh! thank you